### ISSUE TITLE
Nanofloxacin not working as intended

### ISSUE THREAD

**Issue by KingofHONK** [d3athrow/vgstation13#28021]
![image](https://user-images.githubusercontent.com/24867955/96168004-23c43a80-0eee-11eb-8235-22cb70a9fe4b.png)

@DeityLink had told me to create an issue a few months ago so that he didnt forget, and  I forgot to make the issue, and skimming github I didnt see an issue/PR on it so I think we both just straight up forgot.

ANYWAYS I went through a virus curing autism phase back in July where I tested Nanofloxacin's viability as a method for curing viruses. Basically my results were as such:

- Nanofloxacin requires more than 4 applications to get from 0 to 80+ in antibodies (due to overdose)
- The time it takes to find/create the pre-requisites to make Nanofloxacin is significantly higher than if you were to just put infected blood samples in a centrifuge. 
- In its current state, its works exactly like an uncapped spaceacilin but slower, more laborious to create, and without enough of a benefit to warrant the cost of production. 

TL;DR Nanofloxacin is not working as intended, and even if it is, it needs a buff.

**Comment by DeityLink** [comment]
And as expected I completely forgot about it as well.

**Comment by DeityLink** [comment]
I mean it being more laborious to create than spaceacillin or vaccines is intended since it's a cure-all, and it shouldn't act as quickly as a vaccine, but it shouldn't require 4 applications. Tweaking the metabolism down should suffice to let it remain long enough in the body to do its job.

**Comment by KingofHONK** [comment]
Its totally fine for it to be harder/more time consuming to create, its just that the payoff was not worth it. 

Wouldn't upping the metabolism just put Nanofloxacin merely on par with Spaceacilin? You literally just centrifuge it and you have the same thing in arguably less time.

**Comment by DeityLink** [comment]
Except it's uncapped so it can cure even high strength diseases in a reasonable time window.

**Comment by DeityLink** [comment]
@KingofHONK Actually I can't reproduce the issue.
I gave that man 1u of nanofloxacin and it still had over half of it left by the time it reached the 95% cap on both antigen
![image](https://user-images.githubusercontent.com/7573912/96193016-84cc2c80-0f47-11eb-9872-540ad0f8bdf9.png)

And it still had a quarter left by the time it reached 100%
![image](https://user-images.githubusercontent.com/7573912/96193045-98779300-0f47-11eb-80e7-ca86d98728fd.png)

Did you just forget to have people lie in beds?

**Comment by KingofHONK** [comment]
Wtf

**Comment by DeityLink** [comment]
Re-open if you observe the issue again.
